aequilibrae/matrix/aequilibrae_matrix.py

```diff
@@ -283,8 +283,8 @@ def get_matrix(self, core: str, copy=False) -> np.ndarray:
 
     def create_from_omx(
         self,
-        file_path: str,
         omx_path: str,
+        file_path: str = None,
```

Comment:
This is actually a breaking change, people who relied on the position of the arguments (us, in our docs) will have the file paths swapped

Reply:
True, but there is not such way out, as the best behaviour is to have it as a memory-only matrix.

I would doubt anyone uses this, however.